graylog2-web-interface/src/components/rules/RuleForm.tsx

```diff
@@ -102,6 +106,8 @@ const RuleForm = ({ create }) => {
         <PipelinesUsingRule create={create} />
 
         <Input id="rule-source-editor" label="Rule source" help="Rule source, see quick reference for more information." error={errorMessage}>
+          {/* TODO: Figure out issue with props */}
```

Comment:
@gally47 any chance to fix this issue too?

Reply:
No straightforward fix so far, SourceCodeEditor is not recognising the prop 'id' when used in a typescript file, when it clearly exists. I will create a separate Issue/PR for this case to investigate more, what do you think @mnin ?

Reply:
I took a closer look, the same problem occurs among others here: https://github.com/Graylog2/graylog2-server/blob/master/graylog2-web-interface/src/components/sidecars/configuration-forms/ConfigurationForm.tsx#L319. This is out of scope for me.